templates/example.tmpl

```diff
@@ -25,6 +39,11 @@
         {{end}}
       </table>
       {{end}}
+      {{if .PrevExample}}
+      <p class="prev">
```

Comment:
I don't want to add previous links right now, let's just introduce the arrow keys in this PR.

Reply:
Sure, no problem